searx/settings.yml

```diff
@@ -1581,6 +1581,13 @@ engines:
     api_site: 'superuser'
     categories: [it, q&a]
 
+  - name: discusspython
+    engine: discourse
+    shortcut: dpy
+    base_url: 'https://discuss.python.org'
+    categories: [it, q&a]
```

Comment:
I think we can better set these two values as defaults in the python module / can be modified in custom settings anyway .. but there should not be the need to set the defaults here in the settings.yml

@allendema can you patch it .. or if it is OK; I can patch it in my review ..